index.html

```diff
@@ -191,10 +191,31 @@ <h3 id="About-me">About me</h3>
                 </div>
             </section>
         </div>
-
-
-        
-     </main>
+        <div class="footer">
+            <section id="contact-form" class="work-section">
+              <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in</p>
+              <form action="https://formspree.io/forms">
```

Comment:
- [ ] Kindly follow the instructions provided in [Formspree](https://formspree.io/html) to submit the contact form as requested by the previous reviewer.